benchmark/inclusion_compositional_field/inclusion_compositional_field.cc

```diff
@@ -0,0 +1,564 @@
+#include <aspect/material_model/simple.h>
+#include <aspect/velocity_boundary_conditions/interface.h>
+#include <aspect/simulator_access.h>
+#include <aspect/global.h>
+
+#include <deal.II/dofs/dof_tools.h>
+#include <deal.II/numerics/data_out.h>
+#include <deal.II/base/quadrature_lib.h>
+#include <deal.II/base/function_lib.h>
+#include <deal.II/numerics/error_estimator.h>
+#include <deal.II/numerics/vector_tools.h>
+
+
+
+namespace aspect
+{
+  /**
+   * This is the "Pure shear/Inclusion" benchmark defined in the following paper:
+   * @code
+   *  @Article{DMGT11,
+   *    author =       {T. Duretz and D. A. May and T. V. Gerya and P. J. Tackley},
+   *    title =        {Discretization errors and free surface stabilization in the
+   *                  finite difference and marker-in-cell method for applied
+   *                  geodynamics: {A} numerical study},
+   *    journal =      {Geochemistry Geophysics Geosystems},
+   *    year =         2011,
+   *    volume =       12,
+   *    pages =        {Q07004/1--26}}
+   * @endcode
+   *
+   * The results are published in Kronbichler, Heister and Bangerth paper.
```

Comment:
Is this actually true? In other words, is the only difference between the `inclusion_fields` benchmark and the `inclusion` benchmark that is already in the repository that this one uses compositional fields instead of the temperature to describe the setup?